runtime/nnapi-header/include/NeuralNetworksEx.h

```diff
@@ -518,7 +518,42 @@ typedef enum {
    * Outputs:
    * * 0: An output tensor.
    */
-  ANEURALNETWORKS_EINSUM_EX = 50037
+  ANEURALNETWORKS_EINSUM_EX = 50037,
+
+  /** Adds two tensors, element-wise.
+   *
+   * Takes two input tensors of identical type and compatible dimensions. The output
+   * is the sum of both input tensors, optionally modified by an activation function.
+   *
+   * Two dimensions are compatible when:
+   *     1. they are equal, or
+   *     2. one of them is 1
+   *
+   * The size of the output is the maximum size along each dimension of the input operands.
+   * It starts with the trailing dimensions, and works its way forward.
+   *
+   * Example:
+   *
+   *     input1.dimension = {4, 1, 2}
+   *     input2.dimension = {5, 4, 3, 1}
+   *     output.dimension = {5, 4, 3, 2}
+   *
+   * Supported tensor types:
+   * * {@link ANEURALNETWORKS_TENSOR_FLOAT32}
+   * * {@link ANEURALNETWORKS_TENSOR_QUANT8_ASYMM}
+   *
+   * Supported tensor rank: up to 4
+   *
+   * Inputs:
+   * * 0: A tensor.
+   * * 1: A tensor of the same type, and compatible dimensions as input0.
+   * * 2: An INT32 value, and has to be one of the {@link FuseCode} values.
+   *      Specifies the activation to invoke on the result of each addition.
+   *
+   * Outputs:
+   * * 0: The sum, a tensor of the same type as input0.
+   */
```

Comment:
It's good to explain why you pass 50038.

Reply:
The enum value 50038 is used in PR #2271. Therefore, 50039 is used in this PR.